lib/linters.js

```diff
@@ -121,9 +121,140 @@ const js = {
 
       Backend scripts and stylesheets for the admin UI must be repackaged in Vue.
     `
+  },
+  methods: {
+    match: /self\.\w+\s*=.*?(function|=>)/g,
+    message: stripIndent`
+      Method definitions should be moved to the new "methods" section of the module,
+      unless you are extending an existing method with the old "super" pattern, in
+      which case they should move to the new "extendMethods" section, with a
+      "_super" argument at the beginning.
+    `,
+    url: 'https://v3.docs.apostrophecms.org/reference/module-api/module-overview.html#methods'
+  },
+  browserCall: {
+    match: /apos\.push\.browserCall/g,
+    message: stripIndent`
+      The "apos.push.browserCall" mechanism does not exist in A3. See the
+      new "enableBrowserData" method and implement or extend the "getBrowserData"
+      method to make data available at page load time in the browser, or
+      add data attributes to your markup, or use the REST API after page load time.
+    `
+  },
+  reqCall: {
+    match: /req\.browserCall/g,
+    message: stripIndent`
+      The "req.browserCall" mechanism does not exist in A3. See the
+      new "enableBrowserData" method and implement or extend the "getBrowserData"
+      method to make data available at page load time in the browser, or
+      add data attributes to your markup, or use the REST API after page load time.
+    `
+  },
+  tagsField: {
+    match: /type:\s*["']tags[".]/g,
+    message: stripIndent`
+      There is no "tags" field type in A3. If you need tags for a particular
+      piece type, add a second piece type for that purpose and create a "relationship"
```

Comment:
"second" could take an extra cycle to work through. "I already have four piece types. What's the second one?" Simply striking that word would be great.

Reply:
Addressed this.